.github/workflows/daily.yml

```diff
@@ -0,0 +1,25 @@
+name: "Daily Maintenance"
+on:
+  schedule:
+    - cron: "0 0 * * *"
+
+jobs:
+  close-issues:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+    steps:
+      - uses: actions/stale@v5
+        with:
+          days-before-issue-stale: 21
+          days-before-issue-close: 7
+          stale-issue-label: "stale"
+          stale-issue-message: "This issue has been marked that it requires additional information or answers to open questions to be able to get resolved. However, there hasn't been any response within the last 3 weeks.\n If this issue still resists, please provide the requested information. This issue will be automatically closed in one week if there is no response."
+          #close-issue-message: "This issue was closed because it has been inactive another week since being marked as stale."
```

Comment:
```suggestion
          close-issue-message: "This issue was closed because it has been inactive for 4 weeks, and still requires information in order to be resolved."
```

Reply:
I had it disabled because the reminder message has basically all the information.
Changing it to your suggestion.